Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -173,6 +173,10 @@ void RenderBox::willBeDestroyed()
             view().unregisterAnchor(*this);
         if (!style().positionTryFallbacks().isEmpty())
             view().unregisterPositionTryBox(*this);
+        if (isOutOfFlowPositioned()) { // Keep this in sync with styleDidChange().
```

Comment:
Instead of the comment, can we just share code?

Reply:
We can make the conditionals a bit more similar, but I think it's good to keep the comment so people don't add conditions that take it out of sync.